.github/recipe.yaml

```diff
@@ -6,10 +6,12 @@ plugins:
   flutter_app_badger: ["wearable-5.5"]
   flutter_secure_storage: ["wearable-5.5", "tv-6.5"]
   flutter_tts: ["wearable-5.5", "tv-6.5"]
+  google_maps_flutter: ["wearable-5.5", "tv-6.5"]
   integration_test: ["wearable-5.5", "tv-6.5"]
   messageport: ["wearable-5.5", "tv-6.5"]
   package_info_plus: ["wearable-5.5", "tv-6.5"]
-  path_provider: ["tv-6.5"]
+  path_provider: ["wearable-5.5", "tv-6.5"]
+  permission_handler: ["wearable-5.5", "tv-6.5"]
```

Comment:
I have a question.
permission_handler is not support tv profile.(Not applicable for TV)
Is it necessary to test?

Reply:
permission_handler_tizen is "technically" supported on TV (⚠️) but no testing should be necessary. Let me fix.